prime-angular-frontend/src/app/core/resources/health-authority-resource.service.ts

```diff
@@ -283,10 +283,10 @@ export class HealthAuthorityResource {
     })
       .pipe(
         map(({
-               healthAuthoritySite,
-               businessHours,
-               remoteUsers
-             }: { healthAuthoritySite: HealthAuthoritySite, businessHours: BusinessDay[], remoteUsers: RemoteUser[] }) => {
+          healthAuthoritySite,
+          businessHours,
+          remoteUsers
+        }: { healthAuthoritySite: HealthAuthoritySite, businessHours: BusinessDay[], remoteUsers: RemoteUser[] }) => {
```

Comment:
@hogmann @mtpultz Is this indentation also wrong? It appears to be normal for me as it's one Tab up. Just double checking in case my editor is doing its thing again.

Reply:
This looks to be correct